src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTable.tsx

```diff
@@ -124,164 +126,127 @@ export const WorkbookEntriesTable = React.memo<WorkbookEntriesTableProps>(
             [dispatch],
         );
 
-        const chunks = useChunkedEntries(entries);
+        const chunks = useChunkedEntries(entries, !scope);
+        const [dashChunk = [], connChunk = [], datasetChunk = [], widgetChunk = []] = chunks;
```

Comment:
Can we use separate `useChunkedEntries` for the main and for scoped tabs?

Use current `useChunkedEntries` in the `WorkbookTabContent` and write a new one for `WorkbookMainTabContent`?

The current logic seems difficult to maintain.